helpers/filterSeats.js

```diff
@@ -21,8 +24,11 @@ module.exports = function(dust) {
     var venueProduct = _.head(params.venueProducts);
     var ticketRates = params.ticketRates;
     var roomRates = params.roomRates;
-    var bandColours = params.bandColours;
-    var sortByInput = params.sortBy;
+    var sortType = params.sortType;
+    var seatLegend = params.seatLegend;
+
+    var bestPrice = {};
+    var currentPrice, topTicket;
```

Comment:
@owennicol I would stick with the conventions used with all the other dust helpers unless you want to raise a jira that refactors them all.
@putvande Unfortunately no transpiler for the-wall-templater-dustjs or the-wall as far as I know.